### DISCUSSION TITLE
Add a new field to designate module vs script

### DISCUSSION THREAD

**Discussion by alainassaf** [jdhitsolutions/PSProjectStatus#7] [Ideas]
I love this concept. In my day-to-day work, I usually am working on multiple scripts and few modules. I'm going to use this module to track various scripts I develop at work, but adding another field to designate whether the project is a script or module would be of use to me. It may be considered redundant, but I'm curious what other's think.

**Comment by jdhitsolutions** [comment]
I have plans to add a `Comments` property which you could use. I can recognize the value of ` ProjectType`. But what would be the possible values? 

+ ScriptModule
+ BinaryModule
+ Script
+ WPF
+ Python
+ Other

Or would a free-form property be better?

**Comment by alainassaf** [comment]
I don't know what all the project types it could be. I personally prefer a limited selection set, but this could be limiting to others. I think a comment field would be sufficient. As to reporting a version for the project. I see where you grab it from a module manifest, but can it manually be updated or read from a git tag?

**Comment by jdhitsolutions** [comment]
You can always manually set a project version. By default, when you run `New-PSProjectStatus` it will use the module manifest version if found. but even then you could specify a different value. You can specify any value you want.

```poweshell
Set-PSProjectStatus -ProjectVersion 1.2.3
```

Or get creative.

```powershell
$v = (git tag -l | select -first 1).replace("v","")
Set-PSProjectStatus -ProjectVersion $v
```

My tags start with v so I'm stripping it off. Remember that the property is trying to create a [version] type.

And you can always manually edit the JSON file.

**Comment by jdhitsolutions** [comment]
I'll be adding a `Comment` property with corresponding parameters in `New-PSProjectStatus` and `Set-PSProjectStatus` in v0.6.0.
